sdk-java/src/main/java/io/littlehorse/sdk/worker/internal/PollThread.java

```diff
@@ -41,55 +37,42 @@ public PollThread(
             ScheduledTaskExecutor taskExecutor) {
         super(threadName);
         this.stub = stub;
-        this.taskDefId = taskDefId;
-        this.taskWorkerId = taskWorkerId;
-        this.taskWorkerVersion = taskWorkerVersion;
-        this.mappings = mappings;
-        this.executable = executable;
-        this.taskMethod = taskMethod;
-        this.taskMethod.setAccessible(true);
-        this.taskExecutor = taskExecutor;
+        this.bootstrapStub = bootstrapStub;
+        this.semaphore = new Semaphore(inflightRequests);
```

Comment:
Yeah, could we rename `semaphore` to `availableInflightRequests` or something more useful?